docs/rules/extensions.md

```diff
@@ -6,10 +6,12 @@ In order to provide a consistent use of file extensions across your code base, t
 
 ## Rule Details
 
-This rule has one option which could be either a string or an object. If it is `"never"` (the default value) the rule forbids the use for any extension. If `"always"` then the rule enforces the use of extensions for all import statements.
+This rule has one option which could be either a string, an object, or both. If it is `"never"` (the default value) the rule forbids the use for any extension. If `"always"` then the rule enforces the use of extensions for all import statements.
```

Comment:
It's not `one option` anymore, is it? It's either one, which is either a string or an object, or two, with first a string then an object. Or am I misunderstanding something?